src/components/a-icon/index.js

```diff
@@ -19,6 +19,10 @@ class AXAIcon extends BaseComponentGlobal {
   constructor() {
     super({ styles, template });
   }
+
+  didRenderCallback() {
+    this.className = 'a-icon__root';
```

Comment:
@LucaMele 
Just added this class, to be selectable by CSS and not to break any existing functionality

Reply:
Though there might be other issues to be fixed regarding this...